torch_patches/.torch_pin

```diff
@@ -0,0 +1 @@
+3149c3a86bae6dcef8bdda539c533edb12e1b35b
```

Comment:
oh the torch_pin actually doesn't go off a commit hash - you'll need to put `#your_PR_number`. e.g. `#76543`

Reply:
I can switch to the PR no! I'm somewhat confident that it works with **the commit hash** too since it failed tests in an expected way when I added it when I specified `.

Reply:
oh welp never mind then :p I always thought it was just based off the PR number but could totally be wrong